website/templates/homepage/memorial.html

```diff
@@ -0,0 +1,44 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% load missionstatic %}
+{% load missiontime %}
+{% load linkify %}
+{% load pauses %}
+
+{% block full_title %}{{ mission.title }} on Spacelog{% endblock %}
+{% block body-class %}memorial{% endblock %}
+
+{% block meta-description %}
+<meta name="description" property="og:description dbo:abstract" content="Memorial site for {{ mission.title }}.">
+{% endblock %}
+
+{% block content %}
+<div class='crest'>
+  <h1>{{ mission.upper_title }} <em>{{ mission.lower_title }}</em></h1>
+  <img src='{% mission_static mission.name "images/homepage/crew.jpg" %}' alt=''>
+  <div>
+    {% for line in mission.copy.narrative %}
+    <p>
+      {{ line }}
+      {% if forloop.last %}
+      <img class='badge' src='{% mission_static mission.name "images/badge.png" %}' alt=''>
+      {% endif %}
+    </p>
+    {% endfor %}
+  </div>
+</div>
+
+{% spaceless %}
+{% for group in people %}
+  {% for person in group.members %}
+    {% cycle "<div class='group'>" "" ""  %}
+    {% ifequal group.view "full" %}
+      {% include "people/_person.html" %}
+    {% else %}
+      {% include "people/_simple_person.html" %}
```

Comment:
We seem to be filtering out any group where `view` is not `"full"` in the view. Do we need this conditional?